select-xamarin-sdk-v2.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash -e -o pipefail
+
+echo "Set SDK"
+
+get_framework_path() {
+  local framework=$1
+  case $framework in
+  mono) echo '/Library/Frameworks/Mono.framework/Versions' ;;
+  ios) echo '/Library/Frameworks/Xamarin.iOS.framework/Versions' ;;
+  android) echo '/Library/Frameworks/Xamarin.Android.framework/Versions' ;;
+  mac) echo '/Library/Frameworks/Xamarin.Mac.framework/Versions' ;;
+  *) ;;
+  esac
+}
+
+set_current_folder() {
+  local framework=$1
+  local version=$2
+
+  echo "framework = $framework, version = $version"
+
+  if [ ! -z ${framework} ]; then
+    local countDigit=$(echo "${version}" | grep -o "\." | grep -c "\.")
+    if [[ ! countDigit -eq 1 ]]; then
+      echo "Wrong framework's versions."
```

Comment:
Error message is not clear enough.
May be 
```
echo "[WARNING] It is not recommended to specify version in "a.b.c.d" format because your pipeline can be broken suddenly in future. Use "a.b" format.
```
Also, we shouldn't fail script if version contains 4 digits. Let's just throw warning.